desktop/npm-helper.js

```diff
@@ -179,6 +179,14 @@ const commands = {
     help: 'Undiff log send',
     code: undiff,
   },
+  'generate-font-project': {
```

Comment:
If icomoon changes their format, this will break? It's okay if that's so, just want to understand limitations